app/workers/expired_bet_worker.rb

```diff
@@ -0,0 +1,22 @@
+class ExpiredBetWorker
+  include Sidekiq::Worker
+
+  def perform
+    recently_expired_bets.find_each do |bet|
+      SearchApiSaver.new(bet).destroy(action: :deactivate)
+    end
+  end
+
+private
+
+  # job runs every hour but checks for bets which expired in the last
+  # 90 minutes - this is to reduce the chance of something being
+  # missed due to jitter in job start time.
+  def recently_expired_bets
+    Bet.where(
+      "expiration_date IS NOT NULL AND expiration_date >= ? AND expiration_date <= ?",
+      90.minutes.ago,
+      Time.zone.now,
+    )
```

Comment:
Might getting all expired and non-deactivated bets be better, in case a job run fails for some reason?

Reply:
Maybe, but unless we're going to start deleting expired bets from the database, then the worker will take longer and longer to run.  If an expired bet is missed, it can be resent to search-api (through the console, or by updating it in the UI, or by reindexing them all)

Reply:
OK no problem. I'm thinking speed shouldn't be a factor since we could add an index and the number of best bets should remain low.